tests/unit_node/child_process_test.ts

```diff
@@ -786,7 +786,7 @@ Deno.test(async function execFileWithUndefinedTimeout() {
   const { promise, resolve, reject } = Promise.withResolvers<void>();
   CP.execFile(
     "git",
-    ["-v"],
+    ["--version"],
```

Comment:
Oh, it's because I was on an old version of git. Regardless, this seems better if it has broader support.